styles/facts.css

```
@@ -0,0 +1,49 @@
+body {
+    background-color: #d0ebcf;
+}
+
+
+main {
+    font-family: 'Lucida Sans', 'Lucida Sans Regular', 'Lucida Grande', 'Lucida Sans Unicode', Geneva, Verdana, sans-serif;
+    color:#0d6589;
+}
+
+/* li {
+    padding-bottom: 30px;
+} */
+
+footer {
+    margin-top: 3em;
+}
+
+/* h2 {
+    margin-bottom: 2em;
+} */
+
+.facts-container {
+    display: flex;
+    justify-content: space-around;
+    padding-left: 0px;
+    list-style-type: none;
+    width: 100%;
+    flex-wrap: wrap;
+    /* height: 100% */
+    /* align-items: stretch; */
```

Let's remove any styling we aren't using anymore:
```suggestion
footer {
    margin-top: 3em;
}

.facts-container {
    display: flex;
    justify-content: space-around;
    padding-left: 0px;
    list-style-type: none;
    width: 100%;
    flex-wrap: wrap;
```